ansible/start_hashistack.yml

```diff
@@ -36,4 +40,8 @@
       register: result
       until: result.status == 200
       retries: 60
-      delay: 1
\ No newline at end of file
+      delay: 1
+
+    - name: Running poststart books
+      include: "{{ item }}"
+      with_fileglob: "/vagrant/playbooks/poststart/*"
```

Comment:
and here.. + moving the files.
```suggestion
      with_fileglob: "/vagrant/conf/vagrant/ansible/playbooks/poststart/*"
```

Reply:
It isn't necessarily configuration though? In poststart I would start terraform for example.

Reply:
I disagree. It is definitly configuration of the `vagrant` part of the repository.

Reply:
@zhenik ?

Reply:
> A couple of minor changes.
> Also add some documentation on how to use this. Including the fact that they will run in lexical order. Maybe change the name from ´example.yml´ to `0-example.yml` to nudge the use of numbers as prefix.
> As for the change of path, we can consolidate around this pattern in a follow up PR for other stuff as long as you set the tone.

Good comments. I will add documentation, but doing it here will cause a conflict at this point, so I'll make a new issue for it. I agree with changing to use numbers as prefix, good way to drive home the lexical order, and easy to understand.

Reply:
It is an ansible playbook for `startup context`. I would suggest 

```
with_fileglob: "/vagrant/startup/ansible/playbooks/poststart/*"
```

## Notes
Also, we use `/etc/ansible`  directory, where we copy `startup.yml`. 
It would be nice to have a standard for the contextual structure of directories for this `hashistack-vagrant` box. 

So far I see 2 contexts: 
- `build context` (paybook.yml) : not possible to override by user
- `runtime/startup context` (startup.yml) : conditional provisioning (extra_vars), user-provided `pre/post` playbooks, etc..

## Just an example from terraform modules documentation
<img width="876" alt="Screenshot 2020-06-25 at 13 30 20" src="https://user-images.githubusercontent.com/15572799/85712044-3240d900-b6e8-11ea-8c87-a38af0b54ef2.png">

Reply:
In my head we already have separated it all into two contexts.
1. bundled with box
2. possible to change by user

I've thought of what's under `/etc/ansible/ ` as bundled with the box, and what's in `/vagrant` as user changeable. This could be made clearer though. This makes more sense to me, because parts of the `runtime` context should be bundled with the box (more specifically starting the hashistack).

Reply:
 I agree with @m88614 statement

Reply:
Ok so I had a talk with Fredrik. I agree (to some extent) that we put it in conf. I am not sure how this will work in the long run but we'll just have to be open to changing it later! I've put it in conf for now.